### ISSUE TITLE
I guess I'll make a pitch for a generic undo.

### ISSUE THREAD

**Issue by MikeRidgway** [lborgman/easy-cap-ed#41]
I know this isn't easy but it might be nice for undo to be an option for reversing whatever the last change a person made to the transcript was.

0.64

**Comment by lborgman** [comment]
Yes, it might be a very good idea. I dropped the idea for a while because of the concurrent editing. I think I will try to put it back on a user by user level. 

Undo and Redo, of course.
